### PR TITLE
Add feature to pass extra args when pushing image

### PR DESCRIPTION
**Pull request by Divyansh42**
If a user wants to pass few extra args to push
command when pushing image, then he can pass
using 'extra-args' input parameter.
This PR adds another input `extra-args` to pass extra arguments.

Fixes: #17 

Signed-off-by: divyansh42 <diagrawa@redhat.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -68,7 +68,15 @@ Refer to the [`podman push`](http://docs.podman.io/en/latest/markdown/podman-man
...
     <td>No</td>
     <td>After copying the image, write the digest of the resulting image to the file. By default, the filename will be determined from the image and tag.
-    The contents of this file are the <code>digest</code> output.
+    The contents of this file are the <code>digest</code> output.</td>
+  </tr>
+
+  <tr>
+    <td>extra-args</td>
+    <td>No</td>
+    <td>Extra args to be passed to push command when pushing image.</td>
```
Comment (tetchel) [inline README.md:77]: "to be passed to `podman push`."

File: action.yml [hunk 2]
```diff
@@ -25,6 +25,9 @@ inputs:
     description: 'Verify TLS certificates when contacting the registry'
     required: false
     default: 'true'
+  extra-args:
+    description: 'Extra args to be passed to push command when pushing image'
```
Comment (tetchel) [inline action.yml:29]: update this one too (not that it matters)

File: src/index.ts [hunk 3]
```diff
@@ -103,6 +104,10 @@ async function run(): Promise<void> {
         registryPath,
     ];
 
+    if (extraArgs) {
+        args.push(extraArgs);
```
Comment (tetchel) [inline src/index.ts:108]: I think the extra-args may be best off split over newlines so the user doesn't have to quote them.

**Review by tetchel — CHANGES_REQUESTED** [review]

**Comment by tetchel** [comment]
can you add some extra_args to a test case?

**Comment by Divyansh42** [comment]
Figured out that, [pull_request_target](https://docs.github.com/en/actions/reference/events-that-trigger-workflows#pull_request_target) is not running on the PR branch.
So here is the test workflow: https://github.com/Divyansh42/push-to-registry/runs/1849255706?check_suite_focus=true#step:10:110

**Review by tetchel — APPROVED** [review]

**Pull request merged**